test/utils/DateUtil.js

```diff
@@ -29,6 +29,12 @@ module.exports = {
     nowInSeconds: function() {
         return moment().toDate().getTime() / 1000;
     },
+    tomorrowInSeconds: function() {
+        return moment(0, "HH").add(1,'days').toDate().getTime() ;
```

Comment:
I don't know if it is a problem, but I think that getTime() returns time in milliseconds. Should it be like in line number 30 (/ 1000) ?

Reply:
added division